### ISSUE TITLE
Add accessibility tweaks and reminders

### ISSUE THREAD

**Issue by SteveBarnett** [electricbookworks/electric-book-gui#54]

**Comment by SteveBarnett** [comment]
- Add [Heydon's Revenge CSS](https://github.com/Heydon/REVENGE.CSS)
- Add missing `fieldset` and `legends` from form groups
- Add better `:focus` styles
- Add missing `type="button"` attributes (and `aria-labelledby` if needed)

**Comment by SteveBarnett** [comment]
Done in https://github.com/electricbookworks/electric-book-gui/issues/57  \o/
